apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/DecentralizedSynchronizerMigrationIntegrationTest.scala

```diff
@@ -511,10 +511,10 @@ class DecentralizedSynchronizerMigrationIntegrationTest
       val walletUserParty = onboardWalletUser(walletClient, validatorBackend)
       walletClient.tap(tapAmount)
       withClueAndLog(s"${validatorBackend.name} has tapped a amulet") {
-        checkWallet(
+        checkWalletAmount(
```

Comment:
merging automation doesn't always run as we don't wait for rounds, so if all we care is about total amulet then we just check that